index.html

```diff
@@ -13,28 +13,154 @@
 
 <body>
   <header>
-    <div class="logo"><a href="#">Narcisse Jemba</a></div>
+    <div class="logo"><a href="#">Narcisse J.</a></div>
     <nav>
       <img src="img/Union.png">
     </nav>
 
   </header>
-  <section class="headline">
+  <section class="headline headline-bg">
     <h1>I’m Alexandra. Glad to see you!</h1>
     <p class="second-text">I’m a software developer! I can help you build a product , feature or website Look through
       some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to
       contact me.</p>
     <div class="third-text">LET’S CONNECT</div>
-     
-      <ul>
-        <li><a href="#"><img src="img/twitter.png"></a></li>
-        <li><a href="#"><img src="img/Linkedin.png"></a></li>
-        <li><a href="#"><img src="img/m.png"></a></li>
-        <li><a href="#"><img src="img/github.png"></a></li>
-        <li><a href="#"><img src="img/fingers.png"></a></li>
-      </ul>
-     
+
+    <ul class="media-icon">
+      <li><a href="#"><img src="img/twitter.png"></a></li>
+      <li><a href="#"><img src="img/Linkedin.png"></a></li>
+      <li><a href="#"><img src="img/m.png"></a></li>
+      <li><a href="#"><img src="img/github.png"></a></li>
+      <li><a href="#"><img src="img/fingers.png"></a></li>
+    </ul>
```

Comment:
[OPTIONAL] 
- [ ] Please remember to always add `alt` attribute to an `img` tag, as mentioned by the previous reviewer it's important to always include `alt` text for all images. The alt text improves the accessibility of a website for screen readers and also shows up on the screen when your images fail to load.